### ISSUE TITLE
Stats Page has "Preparations" in Spanish

### ISSUE THREAD

**Issue by bronwyncombs** [specify/specify7#3684]
**Describe the bug**
The stats page has preparations at "preparaciones"

**To Reproduce**
1. Go to Statistics
2. See error

**Expected behavior**

Should be the English spelling of preparations

**Screenshots**

![Preparaciones SS](https://github.com/specify/specify7/assets/135047322/562a1b62-90c0-474f-86b0-2ee7ab6164a3)

**Desktop:**
 - OS: Windows 10
 - Browser: Edge
 - Specify 7 Version: 7.9-dev

**Database Name:** UOregonPaleo (6.8.01)

**Reported By**

**Additional context**
Database name or any other context about the problem here.

**Comment by grantfitzsimmons** [comment]
Can you recreate this after deleting the stats page app resource on this database? Is it possible that someone manually changed this name?

**Comment by bronwyncombs** [comment]
Can you help me with finding the stats page in app resources? I can't find it :/ 
However, I did check this issue on another database and found the same spelling in coloradopaleo when editing the stats page there. 

![preparaciones colorado database](https://github.com/specify/specify7/assets/135047322/77a39d2e-6677-41ef-8632-082c5df08f05)

**Comment by grantfitzsimmons** [comment]
## Shared Statistics

These are stored in the `CollectionPreferences` app resource:

<img width="1087" alt="image" src="https://github.com/specify/specify7/assets/37256050/ab739a15-578f-4071-a0ed-3a3d2465d2e8">

You can delete this file and it will reset the shared stats page (the one you see above!)


## Private Statistics
It's stored in the `UserPreferences` file, tacked on at the end of the User Preferences 

<img width="1093" alt="image" src="https://github.com/specify/specify7/assets/37256050/784a2caa-31f2-4fde-b144-ea76afaee084">

See the whole `json` object:
```json
{"form":{"preferences":{"printOnSave":{"CollectionObject":false,"Loan":false,"Gift":true},"useCustomForm":[],"autoNumbering":{"CollectionObject":["catalogNumber"]},"enableCarryForward":[]},"general":{"shownTables":[5,10,1,121,9,3,2,65,69,58,4,100]}},"statistics":{"appearance":{"layout":[{"label":"Personal","categories":[{"label":"Curation","items":[{"type":"DefaultStat","pageName":"user","itemName":"collectionObjectsCataloged","categoryName":"holdings","label":"Collection Objects Cataloged","itemType":"QueryStat"},{"type":"DefaultStat","pageName":"user","itemName":"collectionObjectsDetermined","categoryName":"holdings","label":"Collection Objects Determined","itemType":"QueryStat"}]},{"label":"","items":[]}]}]}},"general":{"ui":{"theme":"light"},"appearance":{"accentColor3":"#ffffff","roundedCorners":false}}}
```

The relevant section comes after `"statistics":`:
```json
"statistics":{"appearance":{"layout":[{"label":"Personal","categories":[{"label":"Curation","items":[{"type":"DefaultStat","pageName":"user","itemName":"collectionObjectsCataloged","categoryName":"holdings","label":"Collection Objects Cataloged","itemType":"QueryStat"},{"type":"DefaultStat","pageName":"user","itemName":"collectionObjectsDetermined","categoryName":"holdings","label":"Collection Objects Determined","itemType":"QueryStat"}]},{"label":"","items":[]}]}]}},"general":{"ui":{"theme":"light"}
```

If I delete the value for `statistics`, it will reset.

**Comment by grantfitzsimmons** [comment]
Looks good to me. Are you still seeing this issue?

**Comment by grantfitzsimmons** [comment]
Fixed in #3736
